packages/ckeditor5-editor-multi-root/src/multirooteditor.ts

```diff
@@ -255,7 +253,7 @@ export default class MultiRootEditor extends DataApiMixin( Editor ) {
 	 * @param elementName Element name for the root element in the model. It can be used to set different schema rules for different roots.
 	 * @param isUndoable Whether creating the root can be undone (using the undo feature) or not.
 	 */
-	public addRoot( { rootName, data, elementName }: RootData, isUndoable: boolean ): void {
+	public addRoot( rootName: string, { data = '', elementName = '$root', isUndoable = false } = {} ): void {
```

Comment:
Is the doclet above building correctly while generating API docs?

Reply:
It didn't build well. In docs, the options was `__namedparams` and it didn't have description. I've added a type.